### ISSUE TITLE
Building under 11ty 1.0.2 or 2.0.0

### ISSUE THREAD

**Issue by tarasis** [quinnanya/quinnanya.github.io#2]
Hi

Discovered your repo while trying to see if anyone had ported Minimal Mistakes to 11ty. I noticed that your repos (both main blog, and the eleven-mistakes) would only build under 11ty 0.12.

If you are interested I can send a Pull Request of the changes needed for the blog repo to build under either 1.0.2 or 2.0.0b1. 

It was in the end a fairly minimal amount of changes.

**Comment by quinnanya** [comment]
That would be wonderful, if you wouldn't mind! I was hoping to do more work on eleven-mistakes last year, but life ended up going another direction. I'm optimistic about poking at a few things it's missing (like RSS feeds) but not really sure the likely timeline.

**Comment by tarasis** [comment]
Okay, will do. 

Pull request is being written atm

I must say thank you for showing that porting Minimal Mistakes was possible. I started playing with your repo, took a bunch of your `.eleventy.js` and then went through the process of porting from scratch myself. (The fun with figuring out where data was, but it all helped understand what was going on). In the end I also pinched your pagination code for the front page because I couldn't get paginatior to play ball. 

I got various bits working on mine like previous/next at bottom of articles, the afore mentioned previous/next buttons lacking formatting. Posts now use the wide class for blog posts. Share buttons now appear. Theme is compiled into Minimal Mistakes (I build 11ty site, then compile the sass. Changed main.scss to main.scss.liquid so it compiles in the theme choice)

If you're curious my repo: https://github.com/tarasis/tarasis.net ( or https://git.tarasis.net/tarasis/tarasis.net ) and I thanked you in https://tarasis.net/2023/1/26/site-moved/

Rob
